ssc/common.cpp

```diff
@@ -1009,15 +1021,19 @@ adjustment_factors::adjustment_factors( compute_module *cm, const std::string &p
 //adjustment factors changed from derates to percentages jmf 1/9/15
 bool adjustment_factors::setup(int nsteps, int analysis_period) //nsteps is set to 8760 in this declaration function in common.h
 {
-	ssc_number_t f = m_cm->as_number( m_prefix + ":constant" );
-	f = 1.0 - f / 100.0; //convert from percentage to factor
+    auto& table = m_cm->get_var_table()->lookup(m_prefix)->table;
```

Comment:
@sjanzou This adds "is_assigned" so that scripts are only required to provide the constant adjust. What do you think?